Dropdown/Dropdown.js

```diff
@@ -124,6 +124,8 @@ const DropdownBase = kind({
 	name: 'Dropdown',
 
 	propTypes: /** @lends agate/Dropdown.DropdownBase.prototype */ {
+		adjusteddirection: PropTypes.oneOf(['above', 'below']),
```

Comment:
Please fix prop name
```suggestion
		adjustedDirection: PropTypes.oneOf(['above', 'below']),
```

Reply:
@vJIYEv, Please check what was the error message.

Reply:
This was the error message.
> Warning: React does not recognize the `adjustedDirection` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `adjusteddirection` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Reply:
I fixed the prop name to `adjustedDirection`.
This error occured because I didn't delete `adjustedDirection` prop when it was not used.
I deleted it and this error does not occur anymore.

Reply:
I cannot find the code that you deleted `adjustedDirection` prop. Can you please check it?

Reply:
`adjustedDirection` prop should have deleted in `DropdownList` component.
`Dropdown` component does not pass `adjustedDirection` prop to `DropdownList` component anymore, instead it passes `direction` with adjustedDirection value now.
So you can see the code deleting `direction` prop. (DropdownList.js line 150)
```javascript
delete rest.direction;
```